activities/Calligra.activity/js/player.js

```diff
@@ -8,6 +8,7 @@ var Player = {
 				<canvas id="letter"></canvas>
 			</div>
 			<button id="player-restart" class="player-restart"></button>
+			<button id="player-next-letter" class="player-next" v-on:click="nextItem()">Try next</button>
```

Comment:
Will be better to use an icon instead. For example something like https://github.com/llaske/sugarizer/blob/dev/activities/MindMath.activity/icons/pass-icon.svg